docs/reference/manifest/equivalentaddin.md

```diff
@@ -0,0 +1,42 @@
+---
+title: EquivalentAddin element in the manifest file
+description: ''
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# EquivalentAddin element
+
+specifies backwards compatibility for an equivalent COM add-in, or XLL.
+
+**Add-in type:** Task pane, Custom function
+
+## Syntax
+
+```XML
+<EquivalentAddin>  
+    <ProgID>{progid}</ProgID>  
+    <Type>COM</Type>  
+</EquivalentAddin>  
```

Comment:
Based on [this page](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/methods), I'm guessing this should look like:
```
<EquivalentAddin>
    …
</EquivalentAddin>  
```